cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -232,25 +235,34 @@ func create(ctx context.Context) (err error) {
 	for i := 0; i < masters; i++ {
 		var cfg runtime.Configurator
 
+		nodeReq := provision.NodeRequest{
+			Name:     fmt.Sprintf("%s-master-%d", clusterName, i+1),
+			IP:       ips[i],
+			Memory:   memory,
+			NanoCPUs: nanoCPUs,
+			DiskSize: diskSize,
+		}
+
+		if i == 0 {
+			nodeReq.Ports = []string{"50000:50000/tcp", "6443:6443/tcp"}
+		}
+
 		if withInitNode {
 			if i == 0 {
 				cfg = configBundle.Init()
 			} else {
 				cfg = configBundle.ControlPlane()
 			}
 		} else {
-			cfg = configBundle.ControlPlaneCfg
+			// Any one of the control plane nodes can be the init node, so we use the
+			// init config's content, but change the type to a control plane.
+			configBundle.InitCfg.MachineConfig.MachineType = runtime.MachineTypeControlPlane.String()
```

Comment:
just a nit, but seems like this potentially might look better if we update MachineType for `cfg`, as here we multiple times update bundle, while other changes in the loop are no-op.

Or even we could have a configBundle method/param to return init bundle with control plane type

Reply:
Good point! :thinking: Most of this is a hack until we can remove the init node type. I am tempted to go the easier path here (updating `cfg`) since will all hopefully go away sooner rather than later. But if we feel that we have a need to make the bundle generator more robust, I can do that.

Reply:
agreed, let's take shortcut, we probably don't need to support this for too long if we say that official way is to use bootstrap API

Reply:
I found a better solution. Since control plane type will have to be what init type is today (it is almost the same even today), we can just generate the controlplane type with the init generator and then change the type string. This has the benefit that we can deprecate the init node config type at a later date and all controlplane configs should be compatible.